src/_http/http2Adapter.js

```diff
@@ -3,17 +3,18 @@ var http2 = require('http2')
 var errors = require('./errors')
 var util = require('../_util')
 
-// Destroy session after 1 minute of inactivity.
-var DESTROY_HTTP2_SESSION_TIME = 1000 * 60
 var STREAM_PREFIX = 'stream::'
 
 /**
  * Http client adapter built around NodeJS http2 module.
  *
  * @constructor
+ * @param {object} options Http2Adapter options.
+ * @param {number} options.http2SessionIdleTime The time an HTTP2 session
+ * may live when there's no activity.
```

Comment:
add `(in milliseconds)`
```suggestion
 * @param {number} options.http2SessionIdleTime The time (in milliseconds) that 
 * an HTTP2 session may live when there's no activity.
```